README.org

```diff
@@ -399,6 +411,8 @@ General is flexible in allowing you to choose how you write things, so if the ab
 (my-normal-and-insert-define-key "/" 'swiper)
 #+end_src
 
+If you will be creating a definer and making a lot of keymaps it is recommended you use a prefix keymap instead of just the prefix keywords for performance reasons (see [[#will-generalel-slow-my-initialization-time][Will general.el slow my initialization time?]]).
```

Comment:
An example here would be worth a lot, like the same one you gave me:

``` elisp
;; this will bind the prefixes to the prefix map
(general-define-key
 :states '(emacs normal visual insert)
 :prefix-map 'my-prefix-map
 :global-prefix "C-c"
 :non-normal-prefix "M-SPC"
 :prefix "SPC")

(general-create-definer my-map
  :keymaps 'my-prefix-map)
```

Reply:
Ah I see you give this example below, so never mind, unless you think it's also worthwhile to include here for locality.

Reply:
Mentioned that there is an example at that link